torch/_dynamo/variables/user_defined.py

```diff
@@ -291,9 +291,16 @@ def call_function(
             and len(kwargs) == 0  # TODO(ybliang): support kwargs
         ):
             unwrapped_args = [x.as_python_constant() for x in args]
-            return GenericContextWrappingVariable(
-                unwrapped_args,
-                cm_obj=self.value(*unwrapped_args),
+            if self.value.__init__ == object.__init__:
+                return GenericContextWrappingVariable(
+                    unwrapped_args,
+                    cm_obj=self.value(*unwrapped_args),
+                )
+
+            return tx.inline_user_function_return(
+                variables.UserFunctionVariable(self.value.__init__, source=self.source),
```

Comment:
Why is this right? `__init__` doesn't even return anything typically

Reply:
It produces a ConstantVariable(NoneType), which iirc, is the right the right thin to push onto the stack.